evap/staff/tools.py

```diff
@@ -292,3 +301,21 @@ def find_next_unreviewed_evaluation(semester, excluded):
         .filter(contributions__textanswer_set__state=TextAnswer.State.NOT_REVIEWED) \
         .annotate(num_unreviewed_textanswers=Count("contributions__textanswer_set")) \
         .order_by('vote_end_date', '-num_unreviewed_textanswers').first()
+
+
+def remove_user_from_represented_and_ccing_users(user, ignored_users=None, test_run=False):
+    remove_messages = []
+    ignored_users = list() if ignored_users is None else ignored_users
+    for represented_user in user.represented_users.exclude(id__in=[user.id for user in ignored_users]):
+        if test_run:
+            remove_messages.append(_("%s will be removed from the delegates of %s.") % (user.full_name, represented_user.full_name))
```

Comment:
We usually the the more recent .format() syntax of python that gets rid of some of the weirdness of the older %-formatting. I'd appreciate if you changed this here (and in the other few cases)